### DISCUSSION TITLE
Add as search engine into Firefox

### DISCUSSION THREAD

**Discussion by UMLAUTaxl** [searchmysite/searchmysite.net#59] [Q&A] [answered]
Hi, I have tried the three dots in the url bar and clicked "Add search engine", then I tried right-clicking at the search field on the site and clicked "Add a keyword for this search ..." and added "sm"  for it. :D But whenever I perform a search it sends me to "403 Forbidden nginx/1.18.0 (Ubuntu)". 
Any idea how I can achieve this?

**Comment by m-i-l** [comment, marked as answer]
The search integration with Firefox was originally working, but unfortunately the mitigation for the large volume of automated SEO searches (https://github.com/searchmysite/searchmysite.net/issues/55) has broken it. (It has also stopped direct links to search results working, which is also unfortunate.)

I'm currently exploring mitigating the SEO searches issue with Cloudflare. If I can get that working I'll be able to remove the reverse proxy config and code change detailed in https://github.com/searchmysite/searchmysite.net/issues/55 , to get the Firefox search (and direct search results links) working once more.
